ipwb/replay.py

```diff
@@ -255,13 +255,33 @@ def show_uri(path, datetime=None):
 
     for idx, hLine in enumerate(hLines):
         k, v = hLine.split(': ', 1)
+
+        if k.lower() == 'transfer-encoding' and v.lower() == 'chunked':
+            resp.set_data(extractResponseFromChunkedData(payload))
         if k.lower() != "content-type":
             k = "X-Archive-Orig-" + k
+
         resp.headers[k] = v
 
     return resp
 
 
+def extractResponseFromChunkedData(data):
+    chunkDescriptor = -1
+    retStr = ''
+
+    (chunkDescriptor, rest) = data.split('\n', 1)
+    while chunkDescriptor.strip() != '0':
+        chunkDecFromHex = int(chunkDescriptor, 16)  # Get dec for slice
```

Comment:
Checking the presence of `;` is not needed, the split will still work if the splitter is not present and return only one item in the list as a whole.